staging_vespalib/src/tests/util/process_memory_stats/process_memory_stats_test.cpp

```diff
@@ -27,7 +27,7 @@ std::string toString(const ProcessMemoryStats &stats)
 
 TEST("Simple stats")
 {
-    ProcessMemoryStats stats(ProcessMemoryStats::create());
+    ProcessMemoryStats stats(ProcessMemoryStats::create(4096));
```

Comment:
The magic number 4096 is used multiple times, please use a named constant expression.